<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientRelationComboBoxPopup</class>
 <widget class="QWidget" name="ClientRelationComboBoxPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabClientRelation">
      <attribute name="title">
       <string>Результат поиска</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CTableView" name="tblClientRelation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearch">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="lblLastName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Фамилия</string>
         </property>
         <property name="buddy">
          <cstring>edtLastName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtLastName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblFirstName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Имя</string>
         </property>
         <property name="buddy">
          <cstring>edtFirstName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtFirstName"/>
       </item>
       <item row="9" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblPatrName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Отчество</string>
         </property>
         <property name="buddy">
          <cstring>edtPatrName</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblBirthDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Дата рождения</string>
         </property>
         <property name="buddy">
          <cstring>edtBirthDate</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="edtPatrName"/>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbDocType"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblDocument">
         <property name="text">
          <string>Документ</string>
         </property>
         <property name="buddy">
          <cstring>cmbDocType</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblSerial">
         <property name="text">
          <string>Серия</string>
         </property>
         <property name="buddy">
          <cstring>edtLeftSerial</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edtLeftSerial"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="edtRightSerial"/>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="edtNumber"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblNumber">
         <property name="text">
          <string>Номер</string>
         </property>
         <property name="buddy">
          <cstring>edtNumber</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="CDateEdit" name="edtBirthDate">
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QPushButton" name="btnFillingFilter">
         <property name="text">
          <string>Автозаполнение данных</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="btnRegistry">
         <property name="text">
          <string>Регистрация</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="edtKey">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;инициалы: ФИО&lt;br/&gt;инициалы и дата рождения: ФИОДДММГГ[ГГ]&lt;br/&gt;сокращение: Фам Им Отч&lt;br/&gt;сокращение и дата рождения: Фам Им Отч ДДММГГ[ГГ]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="chkKey">
         <property name="text">
          <string>Поиск по ключу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnFillingFilter</tabstop>
  <tabstop>edtLastName</tabstop>
  <tabstop>edtFirstName</tabstop>
  <tabstop>edtPatrName</tabstop>
  <tabstop>edtBirthDate</tabstop>
  <tabstop>cmbDocType</tabstop>
  <tabstop>edtLeftSerial</tabstop>
  <tabstop>edtRightSerial</tabstop>
  <tabstop>edtNumber</tabstop>
  <tabstop>chkKey</tabstop>
  <tabstop>edtKey</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tblClientRelation</tabstop>
  <tabstop>btnRegistry</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtKey</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtNumber</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtLeftSerial</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtRightSerial</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>10</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbDocType</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBirthDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtPatrName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFirstName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtLastName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnFillingFilter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
